tutorials/3d/baked_lightmaps.rst

```diff
@@ -157,65 +173,173 @@ can be controlled from the **Bake Mode** menu in lights:
+  shadows casted on baked surfaces.
+- **Default Texels Per Unit:** For meshes that do not specify their own lightmap
+  texel density, this will be used as the value. Higher values result in
+  *lower-resolution* lightmaps, which result in faster bake times and lower file
+  sizes at the cost of blurrier indirect lighting and shadows.
+
+Atlas
+^^^^^
+
+- **Generate:** If enabled, a texture atlas will be generated for the lightmap.
+  This results in more efficient rendering, but is only compatible with the
+  GLES3 rendering backend. Disable this setting if your project is allowed to
+  fall back to GLES2. (This is not the case by default and must be enabled in
+  the Project Settings.) *This setting is ignored when the project is configured
+  to use GLES2 by default.*
+- **Max Size:** The maximum size of the atlas in pixels. Higher values result
+  in a more efficient atlas, but are less compatible with old/low-end hardware.
+  If in doubt, leave this setting on its default value (4096).
+
+Capture
+^^^^^^^
+
+- **Enabled:** This enables probe capture so that dynamic objects can *receive* indirect lighting.
+  Regardless of this setting's value, dynamic objects will not be able to
+  *contribute* indirect lighting to the scene. This is a limitation of lightmaps.
+- **Cell Size:** The distance between lightmap probes in 3D units. Higher values
+  result in more sparse probe placement, which decreases bake times and file
+  size at the cost of lower lighting accuracy for dynamic objects.
+- **Quality:** The lightmap probe generation quality. Higher values result in
+  more accurate lighting, but take longer to bake. This setting does not affect
+  the *density* of the lightmap probes, only their quality.
+- **Propagation:** Similar to :ref:`GIProbe <doc_gi_probes>`'s Propagation property.
+  Higher values result in brighter and more diffuse indirect lighting for
+  dynamic objects. Adjust this value depending on your scene to make dynamic
+  objects better fit with static baked lighting.
+
+Data
+^^^^
+
+- **Light Data**: Contains the light baked data after baking. Textures are saved
+  to disk, but this also contains the capture data for dynamic objects, which can
+  be heavy. If you are using a scene in ``.tscn`` format, you should save this
+  resource to an external binary ``.lmbake`` file to avoid bloating the ``.tscn``
+  scene with binary data encoded in Base64.
+
+The Light Data resource can be edited to adjust two additional properties:
+
+- **Energy:** Adjusts the lightmap's brightness. Higher values result in brighter lightmaps.
+  This can be adjusted at run-time for short-lived dynamic effects such as thunderstorms.
+  However, keep in mind that it will affect *all* baked lights.
+- **Interior:** If enabled, dynamic objects will not make use of environment lighting
+  and will use light probes for ambient lighting exclusively. If disabled, both
+  environment lighting and light probes are used to light up dynamic objects.
```

Comment:
Same as before, voxels *could* be seen as some primitive light probes in a regular grid, but it's not what people generally think when they see the term.